### DISCUSSION TITLE
Feature Request: Uninstallation Instructions for app image and manual install.

### DISCUSSION THREAD

**Discussion by haydonryan** [espanso/espanso#1894] [Feature requests and Ideas]
As a new espanso user, I switched from Autokey because espanso offered "wayland support".  

I'm running Arch linux with Gnome on X11.  Initially I installed using the app image, tested espanso and worked great 
Then i switched to wayland, and got the error message "this is designed for X11 not wayland"
This required an uninstall - but there is no section in the install page https://espanso.org/install/  (or that I could easily find) 
I then followed the instructions for Arch with wayland (manual install).
Tested espanso there, and found I had problems (https://github.com/espanso/espanso/issues/1893) .  Decided to go back to X11, but again no uninstall instructions so just removed the binary.
then installed the x11 version  again.

**Comment by ammo414** [comment]
I believe that the solution for uninstalling appimage is as simple as deleting the appimage -- I did just that and can no longer run espanso commands in the terminal. However, I do agree: I'm not going to try a manual install until I know how to safely uniinstall espanso.

**Comment by clemwo** [comment]
Im on Ubuntu and installed the wrong version. Now before installing the wayland version I wanted to remove the other version but have no clue how. 
Please add uninstall instructions

**Comment by haydonryan** [comment]
I have stopped using espanso entirely.  I haven't tested the latest version, but now I only use wayland.  I don't see anything in the changelogs about fixing those bugs, so I just type manually now.

**Reply by clemwo** [reply]
hm yeah but espanso works great on Windows and Mac. I kinda rely on it now.
